api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -714,15 +738,15 @@ def test_github_webhook_without_installation_id(
     # When
     response = api_client.post(
         path=url,
-        data=WEBHOOK_PAYLOAD_WITHOUT_INSTALLATION_ID,
+        data=WEBHOOK_PAYLOAD_WITH_AN_INVALID_INSTALLATION_ID,
         content_type="application/json",
-        HTTP_X_HUB_SIGNATURE=WEBHOOK_SIGNATURE_WITHOUT_INSTALLATION_ID,
+        HTTP_X_HUB_SIGNATURE=WEBHOOK_SIGNATURE_WITH_AN_INVALID_INSTALLATION_ID,
         HTTP_X_GITHUB_EVENT="installation",
     )
 
     # Then
     mocker_logger.error.assert_called_once_with(
-        "The installation_id is not present in the payload: {'installation': {'test': 765432}, 'action': 'deleted'}"
+        "GitHub Configuration with installation_id 765432 does not exist"
     )
     assert response.status_code == status.HTTP_200_OK
```

Comment:
Same idea with this error, should these really be 200 responses?

Reply:
In this case, 200 is enough.

Reply:
Added